src/js/order.js

```diff
@@ -1,6 +1,11 @@
+/* Copyright(c) 2019 Philip Mulcahy. */
```

Comment:
Not sure I like this territorial copyright thing being so prominent now that others are helping out - let me do some research

Reply:
but no need to change for now